crypto/bls12381_utils.go

```diff
@@ -122,18 +122,14 @@ func randZrStar(x *scalar) {
 	C.bn_randZr_star((*C.bn_st)(x))
 }
 
-// mapToZrStar reads a scalar from a slice of bytes and maps it to Zr
-// the resulting scalar is in the range 0 < k < r
-func mapToZrStar(x *scalar, src []byte) error {
-	if len(src) > maxScalarSize {
-		return invalidInputsErrorf(
-			"input slice length must be less than %d",
-			maxScalarSize)
-	}
-	C.bn_map_to_Zr_star((*C.bn_st)(x),
+// mapToZr reads a scalar from a slice of bytes and maps it to Zr
+// the resulting scalar is in the range 0 <= k < r.
+// It returns true if scalar is zero and false otherwise.
```

Comment:
```suggestion
// mapToZr reads a scalar from a slice of bytes and maps it to Zr.
// The resulting scalar `k` is in the range 0 <= k < r.
// It returns true if scalar is zero and false otherwise.
```